stories/chart.stories.js

```diff
@@ -141,14 +141,16 @@ storiesOf('HURUmap UI|Charts/Bullet Chart', module)
       <BulletChart
         height={number('height', 100)}
         data={object('data', [
-          // When no qualitative measure is given, total would be assumed.
-          // Below data is equivalent to:
-          // [{ x: 12.7, label: 'Living in urban areas' }, { x: 100 }],
-          // [{ x: 9.3, label: 'Living in rural areas' }, { x: 100 }]
-          [{ x: 'Living in urban areas', y: 12.7, unit: '%' }],
-          [{ x: 'Living in rural areas', y: 9.3, unit: '%' }]
+          [
```

Comment:
`unit`? And the comment there I think helps (until we can improve our documentation)